conda.recipe/build.sh

```diff
@@ -8,15 +8,12 @@ BLD_DIR=`pwd`
 SRC_DIR=$RECIPE_DIR/..
 pushd $SRC_DIR
 
-# X.X.X.dev.YYYYMMDD builds
-if [ -e using_tags.txt ]; then
-    version=`git describe --tags`
-else
-    version=`$PYTHON scripts/get_bump_version.py`
-fi
-
-date=`date "+%Y%m%d"`
+version=`$PYTHON scripts/get_bump_version.py`
```

Comment:
Not yet... I planto work in another PR later, and use the TravisCI infrastructure to upload the devel builds on each push... and then we can probably use the TRAVIS_BUILD_COUNT or something like that...